datasets/ar/parliament/crawler.py

```diff
@@ -1,142 +1,125 @@
-    """
-    if person_data.get("name"):
-        person = context.make("Person")
-        person.id = context.make_id(person_data["name"])
-        person.add("name", person_data["name"])
-        person.add("alias", person_data["alias"])
-        person.add("country", "ar")
-        h.apply_name(
-            person,
-            first_name=person_data["first_name"].strip(),
-            last_name=person_data["last_name"].strip(),
-        )
-        person.add("sourceUrl", person_data["url"])
-        person.add("political", person_data["block"])
-
-        position = h.make_position(
-            context,
-            name="Member of National Congress of Argentina",
-            country="ar",
-        )
-        categorisation = categorise(context, position, is_pep=True)
-        if not categorisation.is_pep:
-            return
-        occupancy = h.make_occupancy(
-            context,
-            person,
-            position,
-            False,
-            start_date=person_data["term_start"],
-            end_date=person_data["term_end"],
-            categorisation=categorisation,
-        )
-
-        if not occupancy:
-            return
-        context.emit(person, target=True)
-        context.emit(position)
-        context.emit(occupancy)
+    return profession, birth_date, email
 
 
 def crawl(context: Context):
-    """
-    Main function to crawl and process data from the Argentinian parliament members page.
-    """
-    doc = context.fetch_html(context.dataset.data.url, cache_days=1)
-
-    for element in doc.xpath("//tbody/tr"):
-        person_data = crawl_person(context, element)
-        make_and_emit_person(context, person_data)
+    path = context.fetch_resource("source.html", context.data_url)
+    context.export_resource(path, HTML, title=context.SOURCE_TITLE)
+
+    with open(path, "r") as fh:
+        doc = html.parse(fh)
+
+    # Find the table containing the deputy data
+    table = doc.find(".//table")
+
+    for element in table.findall(".//tr"):
+        # Skip header rows with <th> elements to parse only data rows
+        if element.find(".//th") is not None:
+            continue
+
+        # Initialize the row dictionary that will hold the data for each deputy
+        row = {}
+
+        # Extract cells from the row and map them to columns
+        cells = element.findall(".//td")
+
+        if len(cells) >= 7:
+            row["diputado"] = cells[1].text_content().strip()
+            row["distrito"] = cells[2].text_content().strip()
+            row["mandato"] = cells[3].text_content().strip()
+            row["inicia_mandato"] = cells[4].text_content().strip()
+            row["finaliza_mandato"] = cells[5].text_content().strip()
+            row["bloque"] = cells[6].text_content().strip()
+        else:
+            context.log.warning(
+                "Row does not have enough cells", row_html=html.tostring(element)
+            )
```

Comment:
I think we can go a step further and replace this with h.parse_html_table now that it's in prod.